Automattic/wp-calypso
# Jetpack connect install & activation instructions

This PR starts to implement the changes in the way we tell the user they need to do some manual operations on their wp-admin to keep going with the jetpack-connect flow.

This is the new design:
![image](https://cloud.githubusercontent.com/assets/1554855/14780352/d0dc4098-0adc-11e6-86c1-555c818262aa.png)

And this is how this PR make it looks:

![image](https://cloud.githubusercontent.com/assets/1554855/14780394/001f5796-0add-11e6-8b03-7bbe27f39512.png)

It obviously lacks some design / assets, so I'm passing the ball to @rickybanister  to complete this :) (rick, maybe we can merge this to master as it is right now (maybe hiding those black squares) and work in a new PR, since the examples are going to be a bunch of code and this PR is already a little large)

How to test
========
1. Go to http://calypso.localhost:3000/jetpack/connect 
2. Enter a site where jetpack is not installed
3. You should see the screen above
4. Reload the page and enter a site where jetpack is inactive
4. You should see this:

![image](https://cloud.githubusercontent.com/assets/1554855/14780565/b551e8ea-0add-11e6-9d1f-f58b835de82c.png)

pong @roccotripaldi & @rickybanister
